lib/auto_merger.rb

```diff
@@ -45,10 +45,10 @@ def self.merge_dependabot_prs(dry_run: false)
   end
 
   def self.merge_dependabot_pr(pull_request, policy_manager, dry_run: true)
-    if !policy_manager.is_auto_mergeable?(pull_request)
-      puts "    ...auto-merging is against policy: #{policy_manager.reasons_not_to_merge(pull_request).join(' ')} Skipping."
-    elsif !pull_request.is_auto_mergeable?
+    if !pull_request.is_auto_mergeable?
       puts "    ...auto-merging is allowable in theory, but bad PR: #{pull_request.reasons_not_to_merge.join(' ')} Skipping."
```

Comment:
I'd also tweak the message here, as we haven't checked the policy at this point:

```suggestion
      puts "    ...bad PR: #{pull_request.reasons_not_to_merge.join(' ')} Skipping."
```

Reply:
Done